faros_event.sh

```diff
@@ -6,14 +6,14 @@ version="0.2.0"
 canonical_model_version="0.8.11"
 github_url="https://github.com/faros-ai/faros-events-cli"
 
-declare -a arr=("curl" "jq")
+declare -a arr=("curl" "jq" "sed" "awk")
 for i in "${arr[@]}"; do
     which $i &> /dev/null || 
         { echo "Error: $i is required." && missing_require=1; }
 done
 
 if ((${missing_require:-0})); then
-    echo "Please ensure curl and jq are available before running the script."
+    echo "Please ensure curl, jq, sed, and awk are available before running the script."
```

Comment:
Oh, this is annoying. Perhaps jq can apply lowercase?

Reply:
From what I could figure out, I believe `jq` only supports ascii toLower where as `awk` supports UTF-8 characters as well which made it the better option. Here is a stack overflow that I found that pointed me to a variety of options, but the only one that supports UTF-8 was `awk`.